.gitattributes

```diff
@@ -0,0 +1 @@
+*	-text
```

Comment:
Is it expected that lint should be run on the files in this repository or just web-platform-test? if the former then lint enforces LF so I think what I have is correct if the latter then I think it is ok to limit the change to lint/tests/dummy/*